scripts/build/Platform/Windows/build_config.json

```diff
@@ -355,13 +355,16 @@
     "TAGS": [
       "nightly-clean"
     ],
+    "PIPELINE_ENV":{
+      "NODE_LABEL":"windows-packaging-test"
+    },
```

Comment:
is this temporary?

Reply:
Yes, the jobs have already been defined for this case with a separate AMI